test/project_types/node/commands/populate/customer_test.rb

```diff
@@ -1,16 +1,12 @@
-require 'test_helper'
+# frozen_string_literal: true
+require 'project_types/node/test_helper'
 
 module Node
   module Commands
     module PopulateTests
       class CustomerTest < MiniTest::Test
         include TestHelpers::Schema
 
-        def setup
-          super
-          ShopifyCli::ProjectType.load_type(:node)
-        end
-
```

Comment:
I noticed that the test in this file is still running `run_cmd` are you only updating some of the tests now?